chart/kubeapps/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubeapps
-version: 3.2.3
```

Comment:
we cannot bump the version yet (this would release the `assetsvc` and `asset-syncer`)